ci/check-code-gen.sh

```diff
@@ -17,7 +17,7 @@ PATH=/workspace/gopath/bin:$PATH
 
 set +e
 
-make generated-code -B  > /dev/null
+make generated-code -B
```

Comment:
This is useful output because it's solo-kit, so seeing errors here can be informative